src/functions-and-arrays.js

```diff
@@ -52,7 +106,24 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray() {
+  const uniqueArr = [];
+  const repeatedArr = [];
+  let result;
+  if (wordsUnique.length > 0) {
+    for (let i = 0; i < wordsUnique.length; i++) {
+      if (wordsUnique.indexOf(wordsUnique[i]) !== i && wordsUnique.indexOf(wordsUnique[i]) !== -1) {
```

Comment:
- What exactly does this line of code do? Checking it against the index 
```
wordsUnique.indexOf(wordsUnique[i]) !== i
```

- You actually want to check if you words repeat in the array that you are building. So you take your empty array - initially, and check if the word is already there. If indexOf() is -1, it means it is not there yet and you want to push it inside
```
for (let i = 0; i < wordsUnique.length; i++) {
      if (uniqueArr.indexOf(wordsUnique[i]) === -1) {
          uniqueArr.push(wordsUnique[i]);
      } 
    }
```

- One last note, currently you have return statement inside the for loop -> but actually you want to wait until you check all the words given , and only then return it (so have return after you finish for loop)

Reply:
if there is a repeated element, the second time that it repeats, it will give me the index for the first time it shows up. Therefore, the index will not be equal to i for the specific instance. It's a way to identify repeated elements